x-pack/plugins/stack_alerts/server/feature.ts

```diff
@@ -59,4 +68,31 @@ export const BUILT_IN_ALERTS_FEATURE = {
       ui: [],
     },
   },
+  subFeatures: [
+    {
+      name: 'Manage Alerts',
```

Comment:
I think these need to be i18n?

Reply:
I have these here as examples only. I'm planning on updating the docs to reflect the changes that need to be made and removing the subfeature portion for stack alerts since we are not yet writing alerts as data yet for stack alerts, so no need to give subfeature privileges.